services/clusters-keeper/src/simcore_service_clusters_keeper/core/settings.py

```diff
@@ -162,6 +148,31 @@ class ApplicationSettings(BaseCustomSettings, MixinLoggingSettings):
         auto_default_from_env=True
     )
 
+    CLUSTERS_KEEPER_TASK_INTERVAL: datetime.timedelta = Field(
+        default=datetime.timedelta(seconds=60),
+        description="interval between each clusters clean check (default to seconds, or see https://pydantic-docs.helpmanual.io/usage/types/#datetime-types for string formating)",
+    )
+
+    SERVICE_TRACKING_HEARTBEAT: datetime.timedelta = Field(
+        default=datetime.timedelta(seconds=60),
+        description="Service heartbeat interval (everytime a heartbeat is sent into RabbitMQ)",
+    )
+
+    CLUSTERS_KEEPER_MAX_MISSED_HEARTBEATS_BEFORE_CLUSTER_TERMINATION: NonNegativeInt = Field(
+        default=5,
+        description="Max number of missed heartbeats before a cluster is terminated",
+    )
+
+    CLUSTERS_KEEPER_COMPUTATIONAL_BACKEND_DOCKER_IMAGE_TAG: str = Field(
+        default="master-github-latest",
+        description="defines the image tag to use for the computational backend",
+    )
+
+    CLUSTERS_KEEPER_COMPUTATIONAL_BACKEND_GATEWAY_PASSWORD: SecretStr = Field(
+        default=SecretStr("my_secure_P1ssword"),
```

Comment:
Why a default here?  I would use an optional field (i.e. https://github.com/ITISFoundation/osparc-simcore/blob/7389b713b865579fabb02a655b1e66e65d591401/packages/models-library/src/models_library/utils/pydantic_tools_extension.py#L20


```python
 CLUSTERS_KEEPER_COMPUTATIONAL_BACKEND_GATEWAY_PASSWORD: SecretStr = FieldNotRequired(..., description="If not set, then this feature is disabled"
```

Reply:
as this is not the final. I want to kick out dask-gateway. I think this will disappear. so I will not change it now